README.md

```diff
@@ -2,131 +2,64 @@
 
 Creating XML files of all official D&D sources compatible with Fight Club 5e and Game Master 5e apps for iOS and Android.
 
+## How-to Use This Repository
 
-## Usage
+The files listed in this repository as-is are not compatible with Fight Club 5e. They are instead a collection of individual source files that must be compiled together into a "compendium". That resulting compendium can then be imported into and used by Fight Club 5e.
 
-If you're here to import the sources into your app, these files are not compatible with Fight Club by themselves. Dropbox no longer hosts the compiled compendiums, so it must be built manually with the instructions below or found somewhere else.
+This document makes a distinction between a **compendium** file and a **collection** file. A compendium file is what you ultimately import into Fight Club 5e; it is an XML file that contains all of the source data and is in a format that Fight Club 5e can process. A collection file is the raw source data that exists within this repository, and is not in a format that can be imported into Fight Club 5e. You must first compile a collection file into a compendium file.
 
-This repository is not an application in itself, but you can use it to build your own custom Compendium (see below).
+This repository contains several collection files which can be found within the `Collections` folder. It's worth opening some of those files and noting the data found within them along with their basic structure. Each of those collection files contain entries that point to raw source data file found within the `Sources` folder.
 
+### Download and Extract the Repository to Your Computer
 
-## Contributing
-
-If you'd like to contribute, feel free to fork the repository and submit pull requests with your changes. We are no longer accepting manual changes to the XML source files because these files are now generated from an external source.
+Click on the green "Code" button towards the top of the page, and then click on the "Download ZIP" button on the subsequent modal popup. Extract the ZIP archive to your `Documents` folder. On Windows, this will be `C:\Users\YOUR_USER_NAME\Documents`; on macOS, this will be `/Users/YOUR_USER_NAME/Documents`.
 
+The location on yoru computer that you extract this repository to will be referred to as the **repository root** folder. The path to the repository root should be something like `C:\Users\YOUR_USER_NAME\Documents\FightClub5eXML-master` or `/Users/YOUR_USER_NAME/Documents/FightClub5eXML-master`.
```

Comment:
"your" on this line has been misspelled.

Reply:
Great catch! I've updated the README to fix the spelling error.

Thanks for looking at this.